index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Sandro's portfolio</title>
+</head>
+<body>
+  <div class="general">
+    <header>
+      <div class="mobile">
+        <div id="time">9:41</div>
+        <div id="icons">
+          <img src="./Images/Cellular Connection.png" alt="signal">
+          <img src="./Images/Wifi.png" alt="wifi">
+         <img src="./Images/Battery.png" alt="batery">
+       </div>
+     </div>
+
+     <div class="menu">
+        <p>Jonh Doe</p>
+        <a href="#"><img src="./Images/Union.png" alt="menu"></a>
+     </div>
+   </header>
+  
+   <section class="content">
+      <h1>Hey There. <br> I'm Alex</h1>
+     <h4>I'am a Software Developer</h4>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded<br>don't hestiate to contact me.</p>
+
+      <nav class="media">
+       <a href="https://github.com/sambeck87"><img src="./Images/social/Disabled.png" alt="github"></a>
+       <a href="https://www.linkedin.com/in/sandro-israel-hern%C3%A1ndez-zamora-899386a4/"><img src="./Images/social/Disabled-1.png" alt="linkedin"></a>
+        <a href="#"><img src="./Images/social/Enabled.png" alt=""></a>
+        <a href="https://twitter.com/sambeck4488"><img src="./Images/social/Disabled-2.png" alt=""></a>
+        <a href="#"><img src="./Images/social/Disabled-3.png" alt=""></a>  
+        </nav>
```

Comment:
- [ ] It is required that the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly apply this suggestion to this part of your project.